index.html

```diff
@@ -45,10 +45,246 @@ <h1 class="hero__heading poppins">
+                <li class="poppins">javaScript</li>
+            </ul>
+
+            <div>
+                <button type='button' class="project__button poppins">
+                    see Project
+                </button>
+            </div>
+
+
+
+
+        </div>
+
+
+    </section>
+
+
+    <!-- about us -->
+    <section id="aboutus">
+        <div class="aboutus">
+            <h1 class="aboutus__heading poppins">
+                About<br>
+                Myself
+            </h1>
+            <p class="aboutus__desc poppins">
+                Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.
+            </p>
+            <a class="aboutus__subtitle poppins" href="#">
+                LET’S CONNECT
+            </a>
+            <ul class="aboutus__social">
+                <li> <a class="aboutus__social__link" href="https://twitter.com/ahmed_najash"><img
+                            src="images/Twitter.svg" alt="twitter"></a></li>
+                <li> <a class="aboutus__social__link" href="#"><img src="images/M.svg" alt="M"></a></li>
+                <li> <a class="aboutus__social__link" href="https://www.linkedin.com/in/ahmed-najash-286a671ab/"><img
+                            src="images/Linkedin_icon.svg" alt="icon"></a></li>
+                <li> <a class="aboutus__social__link" href="https://github.com/jash-debug"><img src="images/github.svg"
+                            alt=""></a></li>
+                <li> <a class="aboutus__social__link" href="#"><img src="images/Hand.svg" alt="hand"></a></li>
+            </ul>
+
+            <div>
+                <button type='button' class="aboutus__button poppins">
+                    see Project
+                </button>
+            </div>
+
+            <ul class="about_more">
+                <li class="flex">
+                    <h1 class="aboutus__h1 poppins">Languages</h1>
+                    <img src="/images/aboutus__languages.svg" alt="union">
+                </li>
+
+                <li class="none">
+                    <ul class="about_more">
+                        <li class="flex2">
+                            <img src="/images/Js.svg" alt="union">
+                            <h1 class="aboutmore__h1 poppins">javaScript</h1>
+
+                        </li>
+                        <li class="flex2">
+                            <img src="/images/Css.svg" alt="union">
+                            <h1 class="aboutmore__h1 poppins">HTML</h1>
+
+                        </li>
+                        <li class="flex2">
+                            <img src="/images/html5.svg" alt="union">
+                            <h1 class="aboutmore__h1 poppins">CSS</h1>
+                        </li>
+                    </ul>
+
+                </li>
```

Comment:
Please, make sure that you follow the Figma design in implementing the skills section. Check the above comment for more on why you should follow your given design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185749350-796874d0-7667-4e1a-9a12-1e4fa91f0424.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/185749565-dcbbd24d-5c10-4c2e-ab46-713d50faabbd.png)